src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -209,7 +212,6 @@ protected void onCreate(Bundle savedInstanceState) {
     protected void onStart() {
         Log_OC.v(TAG, "onStart() start");
         super.onStart();
-        getSupportActionBar().setIcon(DisplayUtils.getSeasonalIconId());
```

Comment:
Because with material design you usually do not show an icon on the toolbar/action bar but just a title. Due to the whitespace nature of material you would that have this big white arrow, the icon and the title which doesn't really fit into the bar screen real estate wise.